givitsite/friendreq/tests.py

```diff
@@ -1,3 +1,45 @@
-from django.test import TestCase
+from django.test import TestCase, RequestFactory
+from django.shortcuts import reverse
+from django.contrib.auth.models import User
+from .models import ItemRequest,ItemsFound
+from django.conf import settings
 
-# Create your tests here.
+
+
```

Comment:
Those are part of the test so can be included in the class... also its customary in Python to define constants in UPPERCASE.

Reply:
fixed the const declaration.